buildenv.mk

```diff
@@ -39,6 +39,10 @@ VERSION_STRING := $(vsn_major).$(vsn_minor).$(vsn_patch)
 DATE_STRING := `date "+20%y.%m.%d %k:%M"`
 EHSM_KMS_GIT_SHA=$(shell git rev-parse --short=7 --verify HEAD)
 
```

Comment:
title:
"
Unify the runtime folder for ehsm
"

Reply:
fixed